src/app/components/modal/index.html

```diff
@@ -194,6 +196,9 @@
 				<li>Use a primary <a routerLink="/components/button">button</a> for the most important action the user will take. Use secondary buttons for less important actions. Use a link button for the Cancel action.</li>
 				<li>If a modal has only one action, you should generally use the <a routerLink="/components/confirm">confirm</a> component instead.</li>
 			</ul>
+
+    <div class="sky-subsection-heading">Back button</div>
+      <p>When users open a modal, they should be able to use the back button to close the modal and return to the original context. If users select the back button on a modal with unsaved data, the SPA displays a dialog to confirm that they want to close the modal without saving their work.</p>
```

Comment:
It's minor, but the opening clause seems unnecessary. I'd start this sentence as "Users should be able to..."

Also, I think we can tighten up `If users select the back button on a modal with unsaved data` in the second sentence to `If the modal includes unsaved data`.

Finally, the verb in the first sentence is conditional ("should be able to") while the verb in the second sentence is not ("displays"). It seems like both sentences should either be conditional or not. I mean, "should be able to" suggests that maybe sometimes the back button doesn't work this way. If so, then the second sentence should be conditional to, i.e. "should display." But if the first statement is not actually conditional, then I'd suggest changing "should be able to" to "can."